pywatts/modules/postprocessing/merger.py

```diff
@@ -0,0 +1,95 @@
+from typing import Dict, Union
+
+import numpy as np
+import xarray as xr
+
+from pywatts.core.base import BaseTransformer
+from pywatts.core.exceptions import WrongParameterException
+from pywatts.utils._xarray_time_series_utils import numpy_to_xarray
+
+
+class Merger(BaseTransformer):
+    """
+    The merger reduces a two-dimensional time series with multiple values per time step to a uni-variate time series.
+    The first dimension of the input time series is the time-index. The second dimension looks n-1 steps in the past.
+    I.e., the second dimension contains the following values: x(t - n + 1), x(t - n + 2), .. x(t), where t is the
+    current time-index.
+
+    :param name: The name of the module
+    :type name: str
+    :param method: The method indicates the merging method. Method can be 'mean', 'median', or an integer between 0 and
+                   -1.
+                   For each time step, the 'mean' method takes all values of the input time series that corresponds
+                   to the same time, and calculates the mean.
```

Comment:
For each time step, the 'mean' method takes all values of the input time series that correspond
                   to the same time and calculates the mean.